shared/src/model/notifications.coffee

```diff
@@ -56,11 +56,10 @@ Notifications = {
       @emit('change')
 
   getActive: ->
-    return NOTICES[0] if NOTICES.length
+    active = clone(NOTICES)
```

Comment:
Ah I think I see now... `NOTICES` are set by admins... the poller stuff handles the automated notices... we might want to add a comment explaining this.